backend/requirements-dev.txt

```diff
@@ -1,63 +1,63 @@
 #
-# This file is autogenerated by pip-compile with Python 3.12
+# This file is autogenerated by pip-compile with Python 3.13
 # by the following command:
 #
 #    pip-compile --output-file=requirements-dev.txt --strip-extras requirements-dev.in
 #
 asgiref==3.8.1
     # via
-    #   -r requirements.txt
+    #   -r /var/www/sdu/backend/requirements.txt
```

Comment:
Could you run a `sed -i '' -e "s#/var/www/sdu/backend/##g" requirements-dev.txt` so that we get rid of the absolute path?
(might be `sed -i -e` on Linux but not sure)